handbook/people-ops/offboarding.md

```diff
@@ -10,9 +10,44 @@ Teammate departures will be announced in:
 
 These announcements will be brief and won't go into details to protect the privacy of the departing teammate. If any teammates have additional questions, they can reach out to their manager, the departing teammate's manager, or the @sourcegraph/people-ops team.
 
-## Checklist
+## Offboarding checklist
 
-- [] [Announce the departure](#announcing-departures).
-- [] Remove the teammate from the [team page](../../company/team/index.md).
+- [Announce the departure](#announcing-departures).
+- Remove the teammate from the [team page](../../company/team/index.md).
+- Notify Officengine
```

Comment:
Notify accountants to end payroll or invoice payments and to remove the former teammate from Gusto, Veem, Expensify, Human Interest, and any people-ops systems.